### ISSUE TITLE
IDE ignores documentation hrefs

### ISSUE THREAD

**Issue by rchande** [dotnet/roslyn#5101]
```C#
namespace ClassLibrary1
{
    /// <summary> Follow this link:
    /// <see href="https://microsoft.com"/>
    /// </summary>
    public class Class1
    {
    }
}
```

QuickInfo, SigHelp/MAS for Class1 don't display the link, though it is included in the documentation file generated by the compiler.

**Comment by Pilchie** [comment]
This is a general case of #1516.  Seems like we should just show the content of any attribute in a `<see>` tag, even if we don't do anything special with it.

**Comment by rchande** [comment]
@Pilchie We probably need to fix this for 1.1 as it affects the documentation for various MS SDKs. The content authoring folks are noticing it because they write documentation with links that work on MSDN but not in VS.

**Comment by rchande** [comment]
Closing as dupe.
